Code/GraphMol/ConjugHybrid.cpp

```diff
@@ -116,58 +111,56 @@ namespace MolOps {
 bool atomHasConjugatedBond(const Atom *at) {
   PRECONDITION(at, "bad atom");
 
-  ROMol::OEDGE_ITER beg, end;
-  boost::tie(beg, end) = at->getOwningMol().getAtomBonds(at);
-  while (beg != end) {
-    if (at->getOwningMol()[*beg]->getIsConjugated()) {
+  auto &mol = at->getOwningMol();
+  for (const auto &nbri : boost::make_iterator_range(mol.getAtomBonds(at))) {
+    auto bnd = mol[nbri];
+    if (bnd->getIsConjugated()) {
       return true;
     }
-    beg++;
   }
   return false;
 }
 
 void setConjugation(ROMol &mol) {
   // start with all bonds being marked unconjugated
   // except for aromatic bonds
-  ROMol::BondIterator bi;
-  for (bi = mol.beginBonds(); bi != mol.endBonds(); bi++) {
-    if ((*bi)->getIsAromatic()) {
-      (*bi)->setIsConjugated(true);
+  for (auto bond : mol.bonds()) {
+    if (bond->getIsAromatic()) {
+      bond->setIsConjugated(true);
     } else {
```

Comment:
It looks like the above loop could be simplified as:

```suggestion
for (auto bond : mol.bonds()) {
  bond->setIsConjugated(bond->getIsAromatic());
}
```